core/src/test/java/com/alibaba/fescar/core/protocol/AbstractMessageTest.java

```diff
@@ -0,0 +1,34 @@
+package com.alibaba.fescar.core.protocol;
+
+import com.alibaba.fescar.core.protocol.AbstractMessage;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+
+public class AbstractMessageTest {
+
+  @Test
+  public void bytesToIntInput2OffsetZeroOutputPositive() {
+    final byte[] bytes = {(byte)0, (byte)10};
+    final int offset = 0;
+    final int retval = AbstractMessage.bytesToInt(bytes, offset);
+    Assert.assertEquals(10, retval);
+  }
+
+  @Test
+  public void bytesToIntInput1OffsetNegativeArrayIndexOutOfBoundsException() {
+    final byte[] bytes = {(byte)0};
+    final int offset = -1;
+    thrown.expect(ArrayIndexOutOfBoundsException.class);
+    AbstractMessage.bytesToInt(bytes, offset);
+  }
+
+  @Test
+  public void bytesToIntInput6ZeroOutputZero() {
```

Comment:
Test name looks wrong compared to other tests?
Offset is not referenced in the name, not sure what first "Zero" refers to and the Output is not zero.